pytest_selenium/pytest_selenium.py

```diff
@@ -237,6 +238,14 @@ def pytest_report_header(config, startdir):
 
 @pytest.mark.hookwrapper
 def pytest_runtest_makereport(item, call):
+    from pytest_html import __version__ as PYTEST_HTML_VERSION
+
+    if PYTEST_HTML_VERSION is not None and PYTEST_HTML_VERSION >= "2.1.0":
```

Comment:
Sorry for missing this earlier, you are better off using `disutils.version` for comparison here instead of comparing the strings, as doing so does not produce correct results for all versions.

Is the intention here to change this warning later to also suggest installing a `pytest-html` version with the bug fixed?

Alternatively it would probably be simpler to just fix https://github.com/pytest-dev/pytest-html/issues/308 instead and make a new release, and don't even issue this warning at all? 🤔

Reply:
Yeah, you're right. This was supposed to be a quick fix because it was blocking. I got tunnelvision.

It does however contain some needed fixes, so I'll update.